src/conf_mode/high-availability.py

```diff
@@ -110,6 +110,7 @@ def verify(ha):
                 if tmp in used_vrid_if:
                     raise ConfigError(f'VRID "{vrid}" can only be used once on interface "{interface} with address family IPv4"!')
                 used_vrid_if.append(tmp)
+                ha['vrrp']['group'][group]['ipver'] = '4'
```

Comment:
I don't think that it should be in `verify` section

Reply:
Oh, yes, sorry. I missed this detail, and I also don't think it should be in `verify()`.

Reply:
True, didn't think any further than 'There's already some ipv4 vs ipv6 logic there'. In which section would it fit? get_config?

Reply:
As IP version is determinded from the mandatory `address`  key in `group_config` why not make this even simpler?

Nothing needs to be changed in high-availability.py - it's a simple extension to the Jinja2 template ... something like this below:


```python3
{%         if group_config.rfc3768_compatibility is vyos_defined and group_config.peer_address is vyos_defined %}
    use_vmac {{ group_config.interface }}v{{ group_config.vrid }}v{{ '4' if group_config.address | is_ipv4 else '6' }}
{%             if group_config.peer_address is vyos_defined %}
    vmac_xmit_base
{%             endif %}
{%         endif %}
```

Reply:
@c-po I didn't know you could use is_ipv4, I will change this pull request.